lib/src/resources/app_assets.dart

```diff
@@ -88,4 +88,5 @@ final class AppAssets {
   static const smilingFaceWithHeartEyes =
       'assets/images/smiling-face-with-heart-eyes.gif';
   static const flatImage = 'assets/images/flat_image.png';
+  static const groupe = 'assets/images/Group.png';
```

Comment:
```suggestion
  static const imagesGroup = 'assets/images/group.png';
```
il faut nomer les fichier en minuscule et faire en sorte que le nom du fichier soit en quelques sorte le chemin mais en camelCase (biensur sans asset)